builder.toml

```diff
@@ -68,7 +68,7 @@ version = "0.9.1"
 
 [[buildpacks]]
   id = "salesforce/nodejs-fn"
-  uri = "https://github.com/forcedotcom/nodejs-sf-fx-buildpack/releases/download/v1.9.6/nodejs-sf-fx-buildpack-v1.9.6.tgz"
+  uri = "https://github.com/forcedotcom/nodejs-sf-fx-buildpack/releases/download/v1.9.6/nodejs-sf-fx-buildpack-v1.9.7.tgz"
```

Comment:
`v1.9.7`

Reply:
It'd be cool if there was a test that verified versions in URL.

Reply:
ready for re-review.